src/venia-pwa-concept/index.md

```diff
@@ -5,7 +5,25 @@ title: Venia Theme (Concept)
 The Venia theme is a proof-of-concept Magento theme built using [PWA Buildpack] tools and [Peregrine] components.
 In its current, experimental state, the theme contains examples for Product Details and Category pages.
 
-Visit the [`venia-pwa-concept`] repository to contribute to this project.
+The [`venia-pwa-concept`] project itself contains a module and theme.
+
+## Pwa module
+
+The `module` directory contains the code for a helper module for the Venia theme.
+This module provides the following server-side functionality:
+
+* Renders an app shell with the proper meta information in the `<head />` of each page
+* Assigns a `RootComponent` to an entity or group of entities without requiring a deployment
```

Comment:
Only the 1st of the 3 items in this list is implemented so far. Maybe worth adding a "Coming Soon" or something?